chainer/backends/cuda.py

```diff
@@ -238,7 +238,7 @@ def to_gpu(array, device=None, stream=None):
     """Copies the given CPU array to the specified device.
 
     Args:
-        array (numpy.ndarray, cupy.ndarray, None, list or tuple):
+        array (array, None, list or tuple):
```

Comment:
Doctest seems to be failing because of an ambiguous reference `WARNING: more than one target found for cross-reference 'array': chainer.Parameter.array, chainer.Variable.array`.

Reply:
I couldn't replicate the error due to other non-related failures in my environment.
I hope it's fixed.